phpugms_greeting_2019-03/index.html

```diff
@@ -0,0 +1,251 @@
+                PHPUG Slack (<a href="https://phpug.slack.com/">phpug.slack.com</a>)<br>
+                Get your Invite (<a href="https://php.ug/slackinvite">php.ug/slackinvite</a>)
+            </p>
+        </section>
+
+        <section>
+            <h2>Slide Archive on Github</h2>
+
+
+            <p>
+                Contribute Slides and links on GitHub<br>
+                <a href="https://github.com/phpugms/">github.com/phpugms</a><br>
+                <a href="https://phpugms.github.io/">phpugms.github.io</a>
+            </p>
+        </section>
+
+        <section>
+            <h2>PHP UG Meetings 2019</h2>
+
+            <p>
+                <a href="https://www.meetup.com/phpugms/events/calendar/">meetup.com/phpugms/events/calendar/</a>
+                <a href="https://www.meetup.com/phpugms/events/ical/">meetup.com/phpugms/events/ical/</a>
+            </p>
+
+            <p>Now with rotating weekdays</p>
+
+            <ol style="list-style: none;">
+                <li>Thursday, 2019-04-18</li>
+                <li>Monday, 2019-05-20</li>
+                <li>Tuesday, 2019-06-18</li>
+            </ol>
+        </section>
+
+
+        <section class="l-2up">
+            <h3>CALL FOR PAPERS - ALWAYS OPEN!</h3>
+            <div>
+                <img src="../assets/call-for-papers.png"/>
+            </div>
+
+            <div>
+                ๏ Suggest talk topics<br>
+                ๏ Submit (lightning) talks<br>
+                ๏ Speak together<br>
+                ๏ Preview / Review<br>
+                ๏ Suggest speakers<br>
+                ๏ Your turn!<br>
+            </div>
+        </section>
+
+
+        <section>
+            <h1>Wifi</h1>
+            <p>SSID: CHK24-GUEST<br>Password: y7BHem9htbKP</p>
+        </section>
+
+        <section>
+            <h1>Restroom </h1>
+        </section>
+
+        <section>
+            <h1>Drinks</h1>
+        </section>
+
+        <section>
+            <h2>Thank you sponsors!</h2>
+        </section>
+
+
+        <section>
+            <img src="../assets/check24-logo.png" class="noborder"/>
+            <p>Thanks for Drinks, Wifi and Hosting</p>
+            <p>Looking for a Job? Ask the team!</p>
+        </section>
+
+        <section>
+            <h1>Upcoming Events</h1>
+        </section>
+
+        <section class="l-2up">
+            <h2>WordCamp Osnabr&uuml;ck</h2>
+
+            <strong>Friday/Saturday, March 22/23, 2019</strong><br>
+
+            <p>Nur noch wenige Tickets verf&uuml;gbar</p>
+        </section>
+
+        <section class="l-2up">
+            <h2>PHPUG Münster Meetup 04/2019 - Privacy by design, yes we can!</h2>
+
+            <strong>Tuesday, April 18, 2019</strong><br>
+
+            <p>Michelangelo van Dam (@DragonBe)</p>
+        </section>
+
+        <section>
+            <h2>Todays Speaker</h2>
+
+            <div>
+                <p>Niklas Dz&ouml;sch (@ndzoesch)</p>
+                <img src="img/speaker.jpg">
```

Comment:
Image missing in commit.

Reply:
Added with commit https://github.com/phpugms/phpugms.github.io/commit/450de5202f5a82b3189cdb740e44a726668f25ca.